### DISCUSSION TITLE
Loading Search Engine Page instead of Directly Searching using the desired Search Engine ?

### DISCUSSION THREAD

**Discussion by SheepDomination** [zen-browser/desktop#2040] [General]
Why when typing an address in the address bar, does it load up the search engine page, instead of searching for what was typed into the address bar with the selected search engine within Zen ?

**Comment by ficd0** [comment]
could you please provide a little more detail, I'm afraid I don't really understand what you mean

**Reply by SheepDomination** [reply]
Type something your wanting to search for in the address bar, whatever your Search engine setup is for in Zen. Instead of searching using the selected search engine, it loads the search engine page, making you have to type what your searching, again.

**Reply by ficd0** [reply]
I definitely don't have this behavior. When I press Ctrl+L to focus the address bar and type a query, it takes me straight to the search engine results for my query. In my case (brave search), this is the result: `https://search.brave.com/search?q=test+query+for+github`

Are you using a custom search engine? You may want to double check that you configured it properly, ie. input the correct URL into the Zen settings. Is this also happening with the default search options? (Google, DuckDuckGo, etc.) If everything is configured properly and this is still happening, you may want to submit a bug report.

**Comment by SheepDomination** [comment]
I recently updated too, 1.0.1-a.10 and unlike the version I was using at the time of writing this question. When running CTRL+L the address tab shows, but when searching for something; I'm taken to the search engine page instead of the results ?

I'd love to see a search bar that appears in the middle of the screen instead of going to the search bar.

**Reply by ficd0** [reply]
do you mean that when you search for something and press enter, it takes you to the search engine page with _nothing_ as a query (ie landing page), or the search engine page _with_ the results for your query? Are you saying you would like to see the results displayed directly in the search bar?

**Reply by SheepDomination** [reply]
It takes me to a search engine page with nothing as the query. I would like what I enter into the address bar for searching to load the search engine of choice and show those results.

**Reply by ficd0** [reply]
what search engine are you using, is it custom added?

**Reply by SheepDomination** [reply]
> what search engine are you using, is it custom added?

It's Bing, and it was just selected from the search engine list.

**Reply by nocanoa** [reply]
can u send print from the search engine settings page

**Reply by SheepDomination** [reply]
![zen_u2IcAohLeM](https://github.com/user-attachments/assets/61353b97-0a15-44bd-828c-274c99070908)

**Reply by nocanoa** [reply]
idk u still have the problem but u can add one manualy and add this url in the dialog
> https://www.bing.com/search?q=%s

**Reply by SheepDomination** [reply]
> idk u still have the problem but u can add one manualy and add this url in the dialog
> 
> > https://www.bing.com/search?q=%s

`%s` isn't accept when creating another Bing Search.
